src/main/scala/mesosphere/marathon/core/health/impl/HealthCheckActor.scala

```diff
@@ -132,14 +132,14 @@ private[health] class HealthCheckActor(
     if (consecutiveFailures >= maxFailures && maxFailures > 0) {
       val instanceId = instance.instanceId
       logger.info(
-        s"Detected unhealthy $instanceId of app [${app.id}] version [${app.version}] on host ${instance.agentInfo.host}"
+        s"Detected unhealthy $instanceId of app [${app.id}] version [${app.version}] on host ${instance.hostname}"
       )
 
       // kill the instance, if it is reachable
       if (instance.isUnreachable) {
-        logger.info(s"Instance $instanceId on host ${instance.agentInfo.host} is temporarily unreachable. Performing no kill.")
+        logger.info(s"Instance $instanceId on host ${instance.hostname} is temporarily unreachable. Performing no kill.")
       } else {
-        logger.info(s"Send kill request for $instanceId on host ${instance.agentInfo.host} to driver")
+        logger.info(s"Send kill request for $instanceId on host ${instance.hostname} to driver")
```

Comment:
before the log was printing out hostname, now it will be printing out Some(hostname) I think...

Reply:
let's do `hostname.getOrElse("unknown")` here to be backward compatible and improve readability of the logs :)

Reply:
Good point.